eos-paygd/eos-paygd.service.in

```diff
@@ -1,14 +1,22 @@
 [Unit]
 Description=Pay As You Go Daemon
 Documentation=man:eos-paygd(8)
-Before=systemd-user-sessions.service
+DefaultDependencies=no
+# Only run from initrd
+ConditionPathExists=/etc/initrd-release
+# Don't complete boot without this running
+OnFailure=emergency.target
```

Comment:
iirc emergency.target == drop to root shell.  If a user gets to a root shell without eos-paygd running, we're doomed.

I think this can probably remain as you've written it, but T27043 now becomes more complicated.  I've logged this problem there.

Reply:
Changing this per the comment here: https://phabricator.endlessm.com/T27043#739231